src/blog/2022/07/flowforge-07-released.md

```diff
@@ -0,0 +1,66 @@
+---
+title: FlowForge 0.7 released
+subtitle: Rollbacks Device Environment Variables and a FlowForge Theme
+description: Rollbacks Device Environment Variables and a FlowForge Theme
+date: 2022-07-07 12:00:00.0
+authors: ["sam-machin"]
+---
+Rollback projects to a previous snapshot, Improvements in using Devices and more.
+
+<!--more-->
+[TODO] Update video url and time below.
```

Comment:
Agreed on the video placement. We should remove the generated image then.

Reply:
removing the image for a single post isn't a simple task, we can replace the image but the sizing is fixed so even a 1x1.png leaves a large blank space.
I propose we go with it as-is for this release and before the next one we create a custom template for release posts that sets the embedded YT video in the image space.

Reply:
discussed in https://flowforgeworkspace.slack.com/archives/C02248RCPJ4/p1657096621156179

Reply:
Have raised https://github.com/flowforge/website/pull/148 which adds support for embedding a video in place of the hero image.

Reply:
If/when #148 gets merged, I'll follow-up with the necessary tweaks to this post to make use of it.